content/master/concepts/claims.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Claims
+weight: 60
+---
+
+Claims represents a set of managed resources as a single
+Kubernetes object, inside a namespace. 
+
+Crossplane creates Claims when users access a
+custom API, defined in the CompositeResourceDefinition. 
+
+{{< hint "tip" >}}
+
+Claims are like [composite resources]({{<ref "./composite-resources">}}). The
+difference between Claims and composite resources is Crossplane can create 
+Claims in a namespace, while composite resources are cluster scoped.
+{{< /hint >}}
+
+{{<expand "Confused about Compositions, XRDs, XRs and Claims?" >}}
+Crossplane has four core components that users commonly mix up:
+
+* [Compositions]({{<ref "./compositions">}}) - A template to define how to create resources.
+* [Composite Resource Definition]({{<ref "./composite-resource-definitions">}})
+  (`XRD`) - A custom API specification. 
+* [Composite Resources]({{<ref "./composite-resources">}}) (`XR`) - Created by
+  using the custom API defined in a Composite Resource Definition. XRs use the
+  Composition template to create new managed resources. 
+* Claims (`XRC`) - This page. Like a Composite Resource, but
+  with namespace scoping. 
+{{</expand >}}
+
+## Creating a Claim
+
+Creating a Claim requires a 
+[Composition]({{<ref "./compositions">}}) and a 
+[CompositeResourceDefinition]({{<ref "./composite-resource-definitions">}}) 
+(`XRD`).  
+
+{{<hint "note" >}}
+The XRD must 
+[enable Claims]({{<ref "./composite-resource-definitions#enable-claims">}}).
+{{< /hint >}}
+
+The Composition defines the set of resources to create.  
+The XRD defines the custom API users call to request the set of resources.
+
+![Diagram of the relationship of Crossplane components](/media/composition-how-it-works.svg)
+
+For example, 
+this {{<hover label="xrd1" line="2">}}CompositeResourceDefinition{{</hover>}}
+creates a composite resource API endpoint 
+{{<hover label="xrd1" line="4">}}xmydatabases.example.org{{</hover>}} and
+enables a Claim API endpoint 
+{{<hover label="xrd1" line="11">}}database.example.org{{</hover>}}
+
+```yaml {label="xrd1",copy-lines="none"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata: 
+  name: xmydatabases.example.org
+spec:
+  group: example.org
+  names:
+    kind: xMyDatabase
+    plural: xmydatabases
+  claimNames:
+    kind: Database
+    plural: databases
+  # Removed for brevity
+```
+
+The Claim uses the XRD's 
+{{<hover label="xrd1" line="10">}}claimNames{{</hover>}} to request resources.
```

Comment:
I changed this to "uses the XRD's `kind` API endpoint to request resources"

I'm purposefully vague with "request resources" to avoid unnecessary details about the underlying XR